### ISSUE TITLE
k4aviewer fails to open microphone

### ISSUE THREAD

**Issue by Brent-A** [microsoft/Azure-Kinect-Sensor-SDK#384]
**Describe the bug**
When I try to open a device using k4aviewer, I get an error "Failed to start microphone: unable to open device!"

**To Reproduce**
On my home PC,
Open Azure Kinect Viewer (either 1.0.2 or 1.1.0-beta.0)
Open the device
Click Start
I get the pop-up dialog with "Failed to start microphone: unable to open device!"

**Expected behavior**
No error message, and streaming mic data.

**Logs**
[k4a log.txt](https://github.com/microsoft/Azure-Kinect-Sensor-SDK/files/3229405/k4a.log.txt)

**Screenshots**

k4aviewer:
![Screenshot](https://user-images.githubusercontent.com/24597447/58507455-ab1cdc80-8146-11e9-9733-583c32387b7d.png)

device manager by container:
![Device Manager 1](https://user-images.githubusercontent.com/24597447/58507469-b112bd80-8146-11e9-8d13-095a253c50cd.png)

device manager by connection:
![Device Manager 2](https://user-images.githubusercontent.com/24597447/58507474-b243ea80-8146-11e9-84a8-3cbdd59e0941.png)

**Desktop (please complete the following information):**
 - OS with Version:  Windows 1903 
 - SDK Version:  e.g. 1.1.0-beta.0
 - Firmware version:  
```
[ info     ] : K4A SDK version:     1.1.0-beta.0
[ info     ] : Serial Number:       000070792012
[ info     ] : RGB Sensor Version:  1.6.98
[ info     ] : Depth Sensor Version:1.6.70
[ info     ] : Mic Array Version:   1.6.14
[ info     ] : Sensor Config:       6109.7
[ info     ] : Build type:          Release
[ info     ] : Signature type:      MSFT
```

**Additional context**

**Comment by joylital** [comment]
@Brent-A might be container ID mix up. Try following steps:
- Right clicking on "Azure Kinect Microphone Array", and select "Uninstall device".  (it is the parent device you have selected in above window)
- Detach and reattach the sensor.
- Restart Azure Kinect Viewer and try again

**Comment by Brent-A** [comment]
You can see from my screenshot that it looks to be in the same container as the rest of the Azure Kinect components.

**Comment by xthexder** [comment]
This seems to reproduce on both of my devices as well with the 1.1.0-beta.0 viewer.

**Comment by tesych** [comment]
The issue should be resolved by Windows update.

**Comment by Josephcs88** [comment]
I have updated Windows to 18362.267 and SDK v1.1.1
![image](https://user-images.githubusercontent.com/35250500/62989852-47b86600-be7c-11e9-9592-1773554ce362.png)

But, the same failed message is still there. The following figure was from running Azure Kinect Viewer v1.1.1
![image](https://user-images.githubusercontent.com/35250500/62989790-12ac1380-be7c-11e9-880a-10e248ca073a.png)

When I rebuilt the k4aviewer by gitting clone the source code at GitHub. There is no output from those sub-windows of Infrared, Depth, Color camera, and IMU as follows.
![image](https://user-images.githubusercontent.com/35250500/62990360-4a1bbf80-be7e-11e9-9694-5000f0089849.png)
Did I miss something? 
Your help will be very appreciated.
